python/common/org/python/types/Bool.java

```diff
@@ -34,7 +34,7 @@ void setValue(org.python.Object obj) {
     }
 
     public org.python.Object byValue() {
-        return org.python.types.Bool.getBool(this.value);
+        return this;
     }
```

Comment:
I saw the comment on org.python.Object: "Return a version of the object that can be used when returning by value. For most objects, this will be itself; but primitive types need to return a copy of themselves to ensure that they aren't modified." Which makes sense, but at the same time, there are only 2 `Bool` objects ever, so it seems unneeded.